cleaners/vuze.xml

```diff
@@ -30,7 +33,7 @@
   </option>
   <option id="cache">
     <label>Cache</label>
-    <description>Delete the cache</description>
+    <description>Delete the IPFilter Cache</description>
```

Comment:
The old string is already translated into dozens of  languages. The distinction about IPFilter is not necessary because Vuze has only one kind of cache, and it's also shown in the filename in the preview